gbpservice/nfp/configurator/modules/configurator.py

```diff
@@ -23,6 +24,20 @@
 CONFIGURATOR_RPC_TOPIC = 'configurator'
 LOG = log.getLogger(__name__)
 
+opts = [
```

Comment:
Shall we add these config options in a different file so that configurator module is not changed every time we add new config params.